rust_dev_preview/glue/README.md

```diff
@@ -0,0 +1,67 @@
+# Rust code examples for Glue
+
+## Overview
+
+These examples demonstrate how to perform several Amazon Glue (glue) operations using the developer preview version of the AWS SDK for Rust.
+
+AWS Glue is a serverless data integration service that makes it easier to discover, prepare, move, and integrate data from multiple sources for analytics, machine learning (ML), and application development.
+
+## ⚠ Important
+
+- Running this code might result in charges to your AWS account.
+- Running the tests might result in charges to your AWS account.
+- We recommend that you grant this code least privilege, or at most the minimum permissions required to perform the task. For more information, see [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions are performed throughout the example.
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Prepare a new crawler, configure its job, and ensure an s3 bucket is prepared](src/pepare.rs) (CreateDatabase, CreateCrawler, GetCrawler, GetDatabase, GetTable, CreateJob, GetJob)
+- [Start a new job run, wait for the job run to complete](src/run.rs) (StartJobRun, GetJobRun, ListJobs)
+- [Clean up the resources created for the run](src/cleanup.rs) ()
+
+## Run the examples
+
+### Prerequisites
+
+You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+
+These examples run with the Rust minimum compiler version as supported by the Rust SDK; at the time of this writing, that is Rust 1.61.0. Executables can run from cargo with additional command line arguments documented below and in the binary main functions.
+
+Running these scenarios requires providing an IAM role (as a command line flag) with the [AWSGlueServiceRole](https://us-east-1.console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole$serviceLevelSummary) policy.
+If the role has the name `GlueRole`, you can retrieve the ARN from the AWS CLI in a \*nix environment with `export RUST_SDK_GLUE_TEST_ROLE=$(aws iam get-role --role-name GlueRole --output text --query Role.Arn)`.
```

Comment:
ARN from the AWS CLI --> Amazon Resource Name (ARN) from the AWS Command Line Interface (AWS CLI)